CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 7.1.0 17-June-2019
```

Comment:
Is it planned to release on 6/17? To nitpick, it should be `17-Jun-2019`

Reply:
yep. ⚡️